### PR TITLE
Adds rvm + ruby versions

### PR DESCRIPTION
**Pull request by himanshu0503**
Fixes https://github.com/dry-dock/u14rub/issues/1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM gcr.io/dry-dock/u14:master.5
+ADD . /tmp
+RUN /tmp/script.sh && rm -rf /tmp
+CMD ["/bin/bash"]
```
Comment (pranaypareek) [inline Dockerfile:4]: we can remove the entry point command here

Comment (pranaypareek) [inline Dockerfile:4]: Also, would you please add a line break after each command? Let's keep everything consistent with what we have done with the base images.

File: shippable.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: ruby
+
+install: true
+script: true
```
Comment (pranaypareek) [inline shippable.yml:4]: Would you give a line break here as well please, between install and script?

**Comment by himanshu0503** [comment]
Closing the PR and would be raising a new PR with the `version` format.

**Pull request closed without merging**